Source/AliveLibAO/Paramite.cpp

```diff
@@ -1211,9 +1204,223 @@ void Paramite::State_2_Walking_44B9E0()
     }
 }
 
+int sRunningTable_4BBCC0[14] =
```

Comment:
should be const int, the others should be as well if they're not

Reply:
forgot about that one the others are fine